tests/settings.py

```diff
@@ -62,7 +62,8 @@
 USE_TZ = True
 USE_I18N = False
 USE_L10N = False
-STATIC_URL = '/static/'
+
+STATIC_URL = os.path.abspath('django_freeradius/static') + '/'
```

Comment:
this is wrong. `STATIC_URL` should be a relative URL, not an absolute path. Why have you changed this?
I believe you're doing something wrong if you need to do this.